_docs/quick-start.markdown

```diff
@@ -30,19 +30,66 @@ Click **Create** when ready, and the IDE will generate a project with the follow
 
 {% include h title="Open a project" tag="open-prj" %}
 
-Go to `File | Open` and select the folder containing the root `Cargo.toml` file.
+Go to `File | Open` and select the folder containing the root `Cargo.toml` file. Check the Cargo tool window (`View | Tool windows | Cargo`) to make sure your project was imported successfully:
 
-Check the Cargo tool window (`View | Tool windows | Cargo`) to make sure your project was imported successfully:
+![check-cargo-tw](/assets/quick-start/qsg_cargo_tw.png){:width="300px"}
 
-![check-cargo-tw](/assets/quick-start/qsg_cargo_tw.png){:width="350px"}
-
-{% include h title="Clone a repository" tag="open-prj" %}
+{% include h title="Clone a repository" tag="clone-repo" %}
 
 Go to `VCS | Checkout from Version Control` on the main menu or click **Checkout from Version Control** on the welcome screen:
 
-![qsg_clone](/assets/quick-start/qsg_clone.png){:width="350px"}
+![qsg_clone](/assets/quick-start/qsg_clone.png){:width="300px"}
 
 In the case of GitHub, provide the repository URL, test the connection using the **Test** button, and click **Clone**:
 
-![qsg_clone_dialog](/assets/quick-start/qsg_clone_dialog.png){:width="750px"}
+![qsg_clone_dialog](/assets/quick-start/qsg_clone_dialog.png){:width="700px"}
+
+{% include h title="Configure project settings" tag="settings" %}
+
+In `Settings/Preferences | Languages & Frameworks | Rust`, you can adjust the toolchain settings and select the macro expansion engine.
+
+![check-cargo-tw](/assets/quick-start/qsg_mainsettings.png){:width="750px"}
+
+ If you plan to use an external linter, select it in `Settings/Preferences | Languages & Frameworks | Rust | Cargo`:
+ 
+ ![check-cargo-tw](/assets/quick-start/qsg_linter.png){:width="450px"}
+
+
+{% include h title="Run Cargo commands" tag="run-commands" %}
+
+To run a Cargo command, click ![Run Cargo Command](/assets/quick-start/cargo@2x.svg){:width="16px"} in the Cargo tool window or press `Ctrl` twice. Type the command in the **Run Anything** popup:
+
+![check-cargo-tw](/assets/quick-start/qsg_runanything.png){:width="600px"}
+
+Every time you run a Cargo command, the IDE creates a temporary run/debug configuration. You can save it and create more configurations of the **Cargo Command** type in `Edit Configurations`.
+
+![check-cargo-tw](/assets/quick-start/qsg_configslist.png){:width="200px"}
+
+You can switch Cargo channels (stable, beta, nightly, or dev) separately for each configuration:
+
+![check-cargo-tw](/assets/quick-start/qsg_cargochannel.png){:width="600px"}
+
+{% include h title="Build" tag="build-prj" %}
+
+To build a project, do one of the following:
+- Run `cargo build`
+- Press `Ctrl/⌘ + F9`
+- Use actions from the `Build` menu
+- Run any **Cargo Command** configuration (by default, it includes a build step)
+
+{% include h title="Run" tag="run-prj" %}
+To run an application, choose from these options:
```

Comment:
There is a third option - run the program via gutter icon
And for me it's the most often case how I run the program in IntelliJ via UI so I think it should be at the first place